src/OrchardCore.Modules/OrchardCore.OpenId/AdminMenu.cs

```diff
@@ -20,20 +29,48 @@ public void BuildNavigation(string name, NavigationBuilder builder)
                 return;
             }
 
-            builder
-                .Add(T["Configuration"], configuration => configuration
-                    .Add(T["Security"], "5", security => security
-                        .Add(T["OpenID Connect Apps"], "15", installed => installed
-                            .Action("Index", "Admin", "OrchardCore.OpenId")
-                            .Permission(Permissions.ManageOpenIdApplications)
-                            .LocalNav()
-                        ))
-                    .Add(T["Settings"], settings => settings
-                        .Add(T["OpenID Connect"], "10", entry => entry
-                            .Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = "open id" })
-                            .LocalNav()
-                        ))
-                );
+            builder.Add(T["OpenID Connect"], "15", category =>
```

Comment:
I am ok with it for now, as Workflows does the same thing, but our initial idea with the menu was to keep the existing first level items. Just FYI, no actions necessary.

Reply:
We can certainly revisit that (I went with this option because grouping all the OIDC stuff in the same node seemed clearer and saved one level of depth).

Reply:
It can be grouped, under Configuration. But I don't really care right now.